Explain how to verify the Ubuntu kernel overlay

garrettr opened the pull request:
Resolves #2.

garrettr committed:
Explain how to verify the Ubuntu kernel overlay

Resolves #2.

garrettr commented:
@dolanjs What do you think?

dolanjs commented:
@garrettr I like how you explained verifying the ubuntu and linux kernel sigs. The person that signs the ubuntu kernel will keep changing and will be hard for us to always have the doc up to date. It does seem that most of the ubuntu developers sign each others keys. Which may be the extent that we can really keep verifying versions signed by different people by checking `gpg --list-sigs`

garrettr committed:
Better explanation of using the Web of Trust

Inspired by a comment from @dolanjs [0].

Removes some obsolete text about downloading the key of the
developer that signed the kernel overlay repo (we can no longer
recommend a specific key to download, and have replaced those
instructions with the more general explanation).

[0] https://github.com/freedomofpress/grsec/pull/5#issuecomment-88568565

garrettr committed:
`git checkout` the tag after verifying it
